### PR TITLE
Build instructions patch

### PR DESCRIPTION
**Pull request by lgallindo**
How to properly build and install boron and faun from scratch.

### CODE REVIEW

**Comment by DominusExult** [comment]
In the "sudo apt install..." line you added libvorbis-dev twice.

**Comment by WickedSmoke** [comment]
Hello Lucas,
The Fedora line should be changed to match.  The package there is `pulseaudio-libs-devel`.

**Comment by WickedSmoke** [comment]
The local Boron build could be simplified to this to include the binary:

```
make -C dist boron
make -C dist/boron
sudo make -C dist/boron DESTDIR=/usr install install-dev
```

**Comment by WickedSmoke** [comment]
The local Faun build should leave out `configure` as that will dump the config files in the project root, not src/faun.  It does a release build by default, so this should suffice:
 
```
git submodule init
git submodule update
make -C src/faun
sudo make -C src/faun/ DESTDIR=/usr install
```

**Comment by lgallindo** [comment]
> Hello Lucas, The Fedora line should be changed to match. The package there is `pulseaudio-libs-devel`.

I don't really have a Fedora box to check the equivalent package and test this out, I will try building a VM.

File: doc/build.md [hunk 1]
```diff
@@ -57,18 +57,15 @@ properly configure it for xu4, build `libboron.a`, and install it:
 
     make -C dist boron
     make -C dist/boron libboron.a
```
Comment (WickedSmoke) [inline doc/build.md]: This only builds the library.  Using ` make -C dist/boron` will build both.

File: doc/build.md [hunk 2]
```diff
@@ -57,18 +57,15 @@ properly configure it for xu4, build `libboron.a`, and install it:
...
+    sudo make -C dist/boron DESTDIR=/usr install install-dev
 
 Faun SDKs and source can be downloaded from the
 [Faun Releases](https://github.com/WickedSmoke/faun/releases) page.  The
 source is also available via the src/faun Git submodule:
 
-    git submodule init
-    git submodule update
-    ./src/faun/configure --prefix /usr
+    git submodule update --init --recursive
```
Comment (WickedSmoke) [inline doc/build.md]: The --recursive option isn't needed here (and is misleading about what the repo. contains).

**Comment by WickedSmoke** [comment]
I run Fedora so that package is correct.  You need not trouble yourself with a VM.

**Comment by WickedSmoke** [comment]
If you make those two changes and squash everything into one commit this should be good to go.

**Comment by lgallindo** [comment]
> If you make those two changes and squash everything into one commit this should be good to go.

Two changes and squash done.

**Comment by WickedSmoke** [comment]
@lgallindo, I cleaned up the commit message a bit and pushed to master as 90f0a9240.  Thanks for making the docs more complete!

**Pull request closed without merging**